context/src/main/java/io/micronaut/runtime/converters/time/TimeConverterRegistrar.java

```diff
@@ -162,6 +162,27 @@ public void register(ConversionService<?> conversionService) {
                 temporalConverter
         );
 
+        /*
+         * There's a not-a-bug <a href="https://bugs.openjdk.org/browse/JDK-8069324">JDK-8069324</a>
+         * which unexpectedly fail because offset is mandatory for RFC_1123
+         * To work around this we use special converter for LocalDateTime treating it as UTC timezone
+         */
+        final TypeConverter<LocalDateTime, CharSequence> localDateTimeConverter = (object, targetType, context) -> {
+            try {
+                DateTimeFormatter formatter = resolveFormatter(context);
+                // Treat missing zone as UTC
+                return Optional.of(formatter.format(object.atOffset(ZoneOffset.UTC)));
```

Comment:
Why not `ZoneOffset.systemDefault`?

Reply:
im against any implicit time zone at all, be it UTC or systemDefault. Why does this converter need to support RFC 1123? RFC 1123 makes no sense for LDT, we should not support it just like the JDK doesnt support it.

Reply:
@dstepanov because systemDefault offset on client and server could be different

Reply:
@yawkat I tried to minimize changes without changing the current semantic, especialy it was properly handled in 4.x